aestream/__init__.py

```diff
@@ -11,11 +11,11 @@
     logging.debug("Failed to import Torch: AEStream is running in Numpy mode")
     del logging
 
-from .aestream_ext import FileInput, UDPInput
+from aestream.input import FileInput, UDPInput
```

Comment:
May I suggest renaming 'input' to something like `_input` (and maybe exposing things from it in `__init__.py`). In this particular instance it should be OK because you are just importing some classes, but in principle I think modules should not have the same name as built-ins. Someone might be tempted to do

`from aestream import input`

which will shadow the `input()` function.

Reply:
Hah, that's a great point! I completely missed that. Thanks!